pkg/vetter/conflictingvirtualservicehost/vet.go

```diff
@@ -318,25 +318,34 @@ func addConflictsForSameRoute(trie *routeTrie, conflictingRules [][]routeRule) [
 //   This happens after we've encountered our first "real" route rule. This should
 //   always return false.
 //
-// case 2: There is exactly one regex in the trie
+// case 2: Ancestor and descendant are in the same virtual service:
```

Comment:
Idk if I pointed this out the first time around, but I appreciate how well you commented these cases!